src/pages/EntityDetails/Model/Model.tsx

```diff
@@ -148,13 +152,15 @@ const Model = () => {
                 modelInfoData["cloud"],
                 modelInfoData["cloud-region"]
               ),
+              owner: modelInfoData.owner,
+              credential,
```

Comment:
I added these two bits of info to the model details page so that when using the app on mobile this data is still shown somewhere, as I changed it to hide those columns on some of the tables on small screens.

Reply:
Nice addition!